docs/diagrams/AttendanceRetrievalSequenceDiagram.puml

```diff
@@ -0,0 +1,32 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor black
+skinparam classBackgroundColor LOGIC_COLOR_T1
+skinparam fontColor black
+
+box Attendance Retrieval LOGIC_COLOR_T1
+participant "<<Class>>\nModuleClassUtil" as ModuleClassUtil LOGIC_COLOR
+participant ":ModuleClass" as ModuleClass LOGIC_COLOR
+participant "<<Class>>\nLessonUtil" as LessonUtil LOGIC_COLOR
+participant ":Lesson" as Lesson LOGIC_COLOR
+participant "<<Class>>\nAttendanceRecordListUtil" as AttendanceRecordListUtil LOGIC_COLOR
```

Comment:
Based on the textbook, there seems to be a colon before the class name even for calls to static methods.

```suggestion
participant "<<Class>>\n:ModuleClassUtil" as ModuleClassUtil LOGIC_COLOR
participant ":ModuleClass" as ModuleClass LOGIC_COLOR
participant "<<Class>>\n:LessonUtil" as LessonUtil LOGIC_COLOR
participant ":Lesson" as Lesson LOGIC_COLOR
participant "<<Class>>\n:AttendanceRecordListUtil" as AttendanceRecordListUtil LOGIC_COLOR
```